package.json

```diff
@@ -83,6 +83,7 @@
     "@playcanvas/eslint-config": "^1.3.0",
     "@playcanvas/jsdoc-template": "^1.0.27",
     "@rollup/plugin-babel": "^5.3.1",
+    "@rollup/plugin-node-resolve": "^15.0.0",
```

Comment:
so this allows us to use this locally?
` import { } from 'playcanvas';`

Reply:
this was needed for rollup to find fflate and embed it in the es5 build of extras.

Reply:
we've marked 'playcanvas' as an external module so rollup leaves it alone and the embedding app is responsible for providing it.

in the es5 build playcanvas is renamed to global 'pc' so it all works out magically.